HomeWork_10/TestPing.java

```diff
@@ -0,0 +1,20 @@
+package com.company;
+
+import java.util.Random;
+
+/**
+ * Class that simulates poll of server by assign random number from 10 to 500 ms to it.
+ *
+ * @author Nikita Zenchik
+ */
+public class TestPing {
+
+    /**
+     *  * Simulates poll of received server.
+     *
+     * @return random number from 10 to 500.
+     */
+    public int writePing() {
```

Comment:
"write" supposes the method actually writes something but it doesn't. It confuses